securedrop/tests/functional/journalist_navigation_steps.py

```diff
@@ -640,14 +641,13 @@ def _journalist_downloads_message(self):
 
         # Downloading files with Selenium is tricky because it cannot automate
         # the browser's file download dialog. We can directly request the file
-        # using urllib2, but we need to pass the cookies for the logged in user
+        # using requests, but we need to pass the cookies for the logged in user
         # for Flask to allow this.
         def cookie_string_from_selenium_cookies(cookies):
```

Comment:
Nit: this function name is now misleading so we should update it since now it's returning a dict for `requests`